dali/aux/optical_flow/turing_of/optical_flow_turing.cu

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) 2017-2018, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "dali/aux/optical_flow/turing_of/optical_flow_turing.h"
+
+namespace dali {
+namespace optical_flow {
+namespace kernel {
+
+constexpr size_t kFractionLength = 5;
+constexpr size_t kBlockSize = 256;
```

Comment:
This constant shouldn't be in global scope. Block size is not something universal - I bet this one is bound to one particular kernel. By the way, for the optical flow conversion from fixed fp16 to float, this block size is probably way too small.

Reply:
Done